packages/flutter/lib/src/animation/tween_sequence.dart

```diff
@@ -20,7 +20,7 @@ import 'tween.dart';
 /// 10.0 for the final 40%:
 ///
 /// ```dart
-/// final Animation<double> = TweenSequence(
+/// final Animation<double> tweenSequence = TweenSequence(
```

Comment:
The returned value is just an animation (not a TweenSequence) so it would be best if the variable was `animation`.

Reply:
done